lib/facter/extlib.rb

```diff
@@ -0,0 +1,19 @@
+# All extlib facts are 'namespaced'.  This isn't natively supported by facter, so we do a bit of magic here.
```

Comment:
`s/magic/making this hard to understand and debug`

jokes aside, this is different from all other facts which while having a namespace in this fashion might make sense. it would be way easier for everyone to understand, debug, work with in the future, etc if it worked how other facts worked. It is not as elegant, but just putting `extlib_` in front of the fact works fine.